### PR TITLE
Feature/e UI 8713

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8713

### Change description ###

Regulatory Organisation Type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — DISMISSED** [review]

File: api/openRoutes.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import getConfigurationUIRouter from './configurationUI';
 import getConfigValue from './configValueRouter';
 
 import getLovRefDataRouter from './prd/lov';
+import getRegulatoryOrganisationTypesRouter from './organisationTypesRouter';
 
 // TODO: rename from prdRouter
 import getappInsightsInstrumentationKey from './monitoring-tools';
```
Comment (connorpgpmcelroy) [inline api/openRoutes.ts:9]: minor: import ordering incorrect

File: src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <a class="govuk-back-link" href="javascript:history.back()">Back</a>
+        <span class="govuk-caption-xl">Register an organisation</span>
+        <h1 class="govuk-heading-xl">
+          Who is your organisation registered with?
+        </h1>
+        <hr>
+        <div class="govuk-body govuk-!-margin-top-6">
+           <fieldset class="govuk-fieldset">
+              <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                <div class="govuk-hint">Give as much detail as you can on the regulator or profressional body your organisation is registered with.</div>
```
Comment (connorpgpmcelroy) [inline src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.html:13]: profressional

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by DavidJayakumar — DISMISSED** [review]

File: src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.html (hunk 2 above)
Comment (udaydenduluri33) [reply src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.html:13]: Done, Thanks

File: api/openRoutes.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/openRoutes.ts:9]: Changed, thanks

File: src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { RegisterComponent } from '../../containers/register/register-org.component';
+import { FormControl, FormGroup } from '@angular/forms';
+import { RegisterOrgService } from '../../services/index';
+import { LovRefDataService } from '../../../shared/services/lov-ref-data.service';
+import { RegulatoryOrganisationType } from '../../../shared/models/lovRefData.model';
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts:8]: Fixed it, Thanks

[the remark this answers is not in the document]
Comment (connorpgpmcelroy) [reply src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts:7]: I can still see the incorrect ordering?

File: src/register-org/register-org.routing.ts [hunk 4]
```diff
@@ -13,6 +13,7 @@ import { PaymentByAccountDetailsComponent } from './components/payment-by-accoun
 import { PaymentByAccountComponent } from './components/payment-by-account/payment-by-account.component';
 import { RegisteredWithRegulatorComponent } from './components/registered-with-regulator/registered-with-regulator.component';
 import { RegisterOrgModule } from './register-org.module';
+import { RegulatoryOrganisationTypeComponent } from './components/regulatory-organisation-type/regulatory-organisation-type.component';
```
Comment (connorpgpmcelroy) [inline src/register-org/register-org.routing.ts:16]: Again, minor, but ./components imports should be together

**Review by connorpgpmcelroy — APPROVED** [review]
Have approved since issues are minor but still import ordering issues

**Review by DavidJayakumar — APPROVED** [review]

**Pull request merged**